go/engine/login.go

```diff
@@ -38,18 +38,18 @@ type Login struct {
 }
 
 // NewLogin creates a Login engine.  username is optional.
-// deviceType should be libkb.DeviceTypeDesktop or
-// libkb.DeviceTypeMobile.
-func NewLogin(g *libkb.GlobalContext, deviceType string, username string, ct keybase1.ClientType) *Login {
+// deviceType should be keybase1.DeviceTypeV2_DESKTOP or
```

Comment:
this comment feels unnecessary now that this type exists. Unless you think someone will try to call this with a paper key and that doesn't work?

Reply:
maybe. I didn't look into the details of what would happen using other device types. So I will leave it, it doesn't hurt.